### ISSUE TITLE
Broken links 

### ISSUE THREAD

**Issue by Rajakavitha1** [LiskArchive/lisk-legacy-docs#364]
Pages affected: 
- https://lisk.io/documentation/lisk-sdk/2.0/lisk-elements/packages/api-client/accounts.html
- https://lisk.io/documentation/lisk-sdk/2.0/lisk-elements/packages/api-client/blocks.html
- https://lisk.io/documentation/lisk-sdk/2.0/lisk-elements/packages/api-client/dapps.html
- https://lisk.io/documentation/lisk-sdk/2.0/lisk-elements/packages/api-client/delegates.html
- https://lisk.io/documentation/lisk-sdk/2.0/lisk-elements/packages/api-client/node.html
- https://lisk.io/documentation/lisk-sdk/2.0/lisk-elements/packages/api-client/peers.html
- https://lisk.io/documentation/lisk-sdk/2.0/lisk-elements/packages/api-client/signatures.html
- https://lisk.io/documentation/lisk-sdk/2.0/lisk-elements/packages/api-client/transactions.html
- https://lisk.io/documentation/lisk-sdk/2.0/lisk-elements/packages/api-client/voters.html
- https://lisk.io/documentation/lisk-sdk/2.0/lisk-elements/packages/api-client/votes.html

The link to Core API documentation xref:2.1.0@lisk-core::user-guide/api.json[Core API documentation] links to the current page in earlier version of the docs it results in a 404.

I do see the file here: https://github.com/LiskHQ/lisk-docs/blob/docs-core/modules/ROOT/pages/api.json

Please provide the correct link to [Core API documentation] and I can update the Xref in a new PR

**Comment by Rajakavitha1** [comment]
Hi @Tschakki !!!
Could you please help with this issue.

**Comment by Tschakki** [comment]
Hi @Rajakavitha1 !

Good catch, but before we can fix the links, this issue needs to be solved first: https://github.com/LiskHQ/lisk-docs/issues/368

**Comment by Tschakki** [comment]
This is fixed in the current release.
